UrbanEngine.Web/Controllers/AboutController.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection; 
+using Microsoft.AspNetCore.Mvc;
+
+namespace UrbanEngine.Web.Controllers {
+    [Route( "api/[controller]" )]
```

Comment:
Per our conversation, the API will be hosted at the subdomain api.urbanengine.org so I don't think the `api/` is needed here.

Reply:
updated